CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1-alpha.1] - 2021-05-17
+
+[Unreleased]: https://github.com/datavaluepeople/truman/compare
```

Comment:
This as you intended? Not sure what the hyperlink is for exactly?

Reply:
hey yeah this is just a stub - once the release is tagged, this can point to compare between HEAD and that tag (more generally, each version number other than the first is a hyperlink to compare between that version and the previous)